hack/jenkins/e2e.sh

```diff
@@ -106,6 +106,22 @@ Addon\
 |multiport\sendpoints\
 "
 
+# release-0.21 GCE skip list
+RELEASE_0_21_GCE_SKIP_TEST_REGEX="\
```

Comment:
How is that different from the point in time snapshot that the release fork provides?